src/refactor_nrepl/ns/prune_dependencies.clj

```diff
@@ -116,18 +118,69 @@
                      (-> pattern re-pattern (re-find ns-name)))
                    (libspec-allowlist/libspec-allowlist)))))
 
-(defn- prune-libspec [symbols-in-file current-ns libspec]
-  (if (libspec-should-never-be-pruned? libspec)
+(defn imports->namespaces [imports]
```

Comment:
All public functions should have a docstring (and ideally an `:added` metadata as well).